README.md

```diff
@@ -171,26 +171,68 @@ pip3 install pyani
 
 The simplest route to obtaining these tools is to use `conda`/`bioconda`:
 
 ```bash
-conda install mummer blast legacy-blast -y
+conda install mummer blast legacy-blast fastani -y
 ```
 
 But they can also be installed by following instructions from the tools' own websites.
 
 - **BLAST+** (for `anib`) [ftp://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/LATEST/](ftp://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/LATEST/)
 - **legacy BLAST** (for `aniblastall`) [ftp://ftp.ncbi.nlm.nih.gov/blast/executables/release/LATEST/](ftp://ftp.ncbi.nlm.nih.gov/blast/executables/release/LATEST/)
 - **MUMmer** (for `anim`) [http://mummer.sourceforge.net/](ftp://ftp.ncbi.nlm.nih.gov/blast/executables/release/LATEST/)
+- **fastANI** (for `fastani`) [https://github.com/ParBLiSS/FastANI](https://github.com/ParBLiSS/FastANI)
 
 ### NOTE: Installing legacy BLAST
 
 **The legacy BLAST executable available from NCBI will not run on macOS Big Sur.**
 
 If you wish to use `pyani blastall` or the `ANIblastall` method with the legacy `pyani` interface, you will require a locally-installed copy of this package. This is one of the packages linked in the `requirements-thirdparty.txt` file.
 
+### Note: Installing fastANI
+
+**`fastANI` installation (via `conda`) will fail for Python 3.9 on macOS.**
+
+If you are using Python ≥3.9 to run `pyani` on macOS, `fastANI` will fail to install due to errors in the `conda` recipe. Using an earlier version of Python may be the simplest solution. However, you can also install `fastANI` yourself—via `conda`—using a slightly altered version of the command produced from the `conda` recipe, as shown below.
+
+*N.B. We have placed the `fastANI` requirement in its own `requirements-fastani.txt` file so that this will not also cause other installations to fail.*
+
+##### Bypassing `conda`
+
+```bash
+c++ -O3 -DNDEBUG -std=c++11 -Isrc \
+-I ${ENV_DIR}/lib/include -mmacosx-version-min=10.7 \
+-stdlib=libc++ -Xpreprocessor -fopenmp -lomp \
+-DUSE_BOOST src/cgi/core_genome_identity.cpp -o fastANI \
+${ENV_DIR}/lib/lib/libboost_math_c99.a \
+-lstdc++ -lz -lm
+```
+
+Two alterations need to be made to the file name specified in the fifth line here:
+
+1. One of the `/lib`s should be removed.
+1. The file extension should be changed from `.a`, to `.dylib`.
+
+The resultant command ought to look something like this (with ${ENV_DIR} being wherever you're installing `fastANI`):
+
+```bash
+c++ -O3 -DNDEBUG -std=c++11 -Isrc \
+-I ${ENV_DIR}/lib/include -mmacosx-version-min=10.7 \
+-stdlib=libc++ -Xpreprocessor -fopenmp -lomp \
+-DUSE_BOOST src/cgi/core_genome_identity.cpp -o fastANI \
+${ENV_DIR}/lib/libboost_math_c99.dylib \
+-lstdc++ -lz -lm
+```
+
+For a more technical overview of the issue, please see https://github.com/widdowquinn/pyani/issues/377.
```

Comment:
Maybe flag explicitly that other solutions might be found on this thread, too.

Reply:
```suggestion
For a more technical overview of the issue, and other solutions that may be shared there, please see https://github.com/widdowquinn/pyani/issues/377.
```